src/hypervisor.c

```diff
@@ -105,7 +105,7 @@ cc_oci_expand_netdev_cmdline(struct cc_oci_config *config, guint index) {
 	return g_strdup("");
 }
 
-#define QEMU_FMT_DEVICE "driver=virtio-net-pci,netdev=%s"
+#define QEMU_FMT_DEVICE "driver=virtio-net-pci,bus=/pci-lite-host/pcie.0,addr=%x,netdev=%s"
```

Comment:
@amshinde may make sense to offset the index. i.e. not start at 0. May even make sense to make the offset configurable.